pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -548,3 +554,17 @@ func upsertObject(ctx context.Context, c ctrlruntimeclient.Client, obj crcontrol
 	}
 	return err
 }
+
+var deniedRegistries = sets.NewString()
```

Comment:
Eh, a global? That's not nice, we should avoid that IMO.

Reply:
Done.